src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -49,6 +49,27 @@ To view the logs for a build, scroll to the build checks at the bottom of the PR
 
 ![Screenshot of the build checks section of the PR page, with arrow pointing to primary link](./images/ci-build-logs.png 'Link to the build details with logs')
 
+## Pacman Package Cache
+
+When buildkite receives a webhook to build and test a PR, the buildkite agents pulls down the code and starts building a docker image based off the Dockerfile in that branch.
```

Comment:
Capitalize Buildkite (like we do GitHub and others) and Docker (like they do in their own documentation). They should only be lowercase in things like commands.

```suggestion
When Buildkite receives a webhook to build and test a PR, the Buildkite agents pulls down the code and starts building a Docker image based off the Dockerfile in that branch.
```

There might be other instances on this page that need updating.